aspnetcore/host-and-deploy/blazor/client-side.md

```diff
@@ -227,6 +227,17 @@ If a *500 - Internal Server Error* is received and IIS Manager throws errors whe
 
 For more information on troubleshooting deployments to IIS, see <xref:host-and-deploy/iis/troubleshoot>.
 
+### Azure Storage
+
+Static file hosting provided by Azure Storage permits serverless Blazor app hosting. Custom domain names, the Azure Content Delivery Network (CDN), and HTTPS are supported.
```

Comment:
is "permit" the right word here?

Reply:
It's ok, but I'll change it. It's not great.